client/src/+assets/profile.tsx

```diff
@@ -95,6 +95,7 @@ export const Profile: NextPage = () => {
     let assetArray = []
     if (!completed) {
       setCompleted(true)
+      if (processing) setProcessing(false)
```

Comment:
this doesn't make a lot of sense semantically. right?

Reply:
yes because only in case that processing is enable will execute the set otherwise will always execute set, it is a optimization execution

Reply:
ok got it :)